packages/core-data/src/entities.ts

```diff
@@ -13,121 +13,173 @@ import { __ } from '@wordpress/i18n';
  * Internal dependencies
  */
 import { addEntities } from './actions';
+import {
+	Attachment,
+	Comment,
+	Context,
+	EntityRecord,
+	MenuLocation,
+	NavigationArea,
+	NavMenu,
+	NavMenuItem,
+	Page,
+	Plugin,
+	Post,
+	Settings,
+	Sidebar,
+	Taxonomy,
+	Theme,
+	Type,
+	Updatable,
+	User,
+	Widget,
+	WidgetType,
+	WpTemplate,
+	WpTemplatePart,
+} from './types';
```

Comment:
fun fact! we can import these altogether and reference them as if they were properties of the module.

```ts
import type * as EntityTypes from './types';

…
| DeclaredEntity< 'root', 'site', EntityTypes.Settings< C > >
```

this can free up the local module's namespace for those same entity kinds

Reply:
Good idea!